exo-go/test_helpers/shared_feature_context.go

```diff
@@ -131,7 +131,7 @@ func SharedFeatureContext(s *godog.Suite) {
 
 	s.Step(`^it prints "([^"]*)" in the terminal$`, func(text string) error {
 		if childCmdPlus != nil {
-			return childCmdPlus.WaitForText(text, time.Minute)
+			return childCmdPlus.WaitForText(text, time.Minute*2)
```

Comment:
Auhh okay. Lets make a new step for this then so that we don't inflate the timeout for other steps. We could probably join those 5 steps into one with is something like
```
Then I eventually see the following snippets:
  | Testing service 'users-service'  |
  | 'users-service' tests passed     |
  | Testing service 'tweets-service' |
  | 'tweets-service' tests passed    |
  | All tests passed                 |
```